harness/determined/cli/workspace.py

```diff
@@ -146,6 +148,51 @@ def list_pools(args: argparse.Namespace) -> None:
     )
 
 
+def list_workspace_namespace_bindings(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_ListWorkspaceNamespaceBindings(sess, id=w.id)
+    if resp.namespaceBindings:
+        print("workspace-namespace bindings for workspace", args.workspace_name)
```

Comment:
Can we use formatted string, it might be easier to see what the final output will look like? Same for other print statements.
```suggestion
        print(f"workspace-namespace bindings for workspace {args.workspace_name}")
```